### PR TITLE
add publish GH workflow, bump version to 1.3.0

### PR DESCRIPTION
**Pull request by pawelmhm**

### CODE REVIEW

**Comment by pawelmhm** [comment]
I'd like to merge https://github.com/scrapy/scrapyd/pull/425 on Monday, and then I think we're good to release 1.3.0. 

There is lots of other work to be done, but it can be done on other versions.

File: docs/news.rst [hunk 1]
```diff
@@ -5,21 +5,28 @@ Release notes
...
+- support for HTTP authentication in scrapyd server
+- replaced deprecated SafeConfigParser with ConfigParser
+- much improved test coverage
+- improved HTTP headers in webservice
 
 Removed
 ~~~~~~~
 
+- Python 2 support
+- Python
```
Comment (jpmckinney) [inline docs/news.rst:29]: Missing content?

Comment (pawelmhm) [reply docs/news.rst:29]: yes sorry, christmas mood will fix

Comment (pawelmhm) [reply docs/news.rst]: I updated news, now just waiting for pypi token and we can release @Gallaecio can you add secret pypi token here?

**Review by jpmckinney — APPROVED** [review]

File: docs/news.rst (hunk 1 above)
Comment (Gallaecio) [reply docs/news.rst]: `PYPI_TOKEN` defined.

**Review by Gallaecio — APPROVED** [review]

**Review by Rotzbua — CHANGES_REQUESTED** [review]
Just a tiny note:
There is an outdated requirement in the docs `Python 2.7` https://github.com/scrapy/scrapyd/blame/9fae00bb76813359af6d709a343df0f46f529eda/docs/install.rst#L15

**Comment by pawelmhm** [comment]
thanks @Rotzbua I fixed that. Just a note we should also remove six from project, but maybe on a separate PR and with next version.

**Pull request merged**